Modules/IO/PhilipsREC/src/itkPhilipsRECImageIO.cxx

```diff
@@ -191,17 +191,16 @@ PhilipsRECImageIOGetImageTypeOffset(int
                                     PhilipsPAR::PARSliceIndexScanSequenceVector sliceScanSequenceIndex)
 {
   int index = volumeIndex * parParam.num_slice_repetitions * numSlices + slice * parParam.num_slice_repetitions;
-  int i;
-
-  for (i = 0; i < parParam.num_slice_repetitions; ++i)
+  int i = 0;
+  for (; i < parParam.num_slice_repetitions; ++i)
   {
     if ((sliceImageTypesIndex[index + i].second == imageType) &&
         (sliceScanSequenceIndex[index + i].second == scanSequence))
     {
       break;
     }
   }
-  return i;
+  s return i;
```

Comment:
`s `? A typo, I gues`s `... does it compile?

Reply:
`PhilipsRECImageIO` is off by default, including in the CI.

Reply:
We probably should at least have one CI that is more aggressive with non-default modules turned on. I'd add DCMTK to that list too.

Reply:
`PhilipsRECImageIO` is small and takes very little time to compile. Mistakes in changes to it in other PRs also recently broke it as I recall...